packages/graph/hash_graph/lib/graph/src/api/rest/link.rs

```diff
@@ -58,12 +58,16 @@ impl RoutedResource for LinkResource {
 }
 
 #[derive(Serialize, Deserialize, Component)]
-#[serde(rename_all = "camelCase")]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
 struct CreateLinkRequest {
     target_entity_id: EntityId,
     #[component(value_type = String)]
     link_type_id: VersionedUri,
     owned_by_id: AccountId,
+    // TODO: Consider if ordering should be exposed on links as they are here. The API consumer
+    //   manages ordering currently.
+    //   https://app.asana.com/0/1202805690238892/1202937382769278/f
+    order: Option<i32>,
```

Comment:
Do you recall the reasoning for naming this `order` instead of `index`? Want to flag this introduces an additional inconsistency with the current BP types (see [the definition for links](https://blockprotocol.org/docs/spec/graph-service#entity-definition))

Reply:
I had initially named this index, but changed it to order last second because I had called it `link_order` in the database. I'll change it back to be consistent with the BP types